catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L4area.jsonnet

```diff
@@ -0,0 +1,168 @@
+    meet global methane mitigation goals.
+
+    Dispersed area emissions are estimated from methane concentration observations 
+    in the form of column-averaged dry-air mole fractions of methane (XCH4) using 
+    an inverse model. An atmospheric transport model - the Stochastic Time-Inverted 
+    Lagrangian Transport (STILT) model; [Lin et al. (2003)](https://doi.org/10.1029/2002JD003161), 
+    [Fasoli et al. (2018)](https://doi.org/10.5194/gmd-11-2813-2018); driven 
+    by meteorological data from the National Centers for Environmental 
+    Prediction ([NCEP](https://www.weather.gov/ncep/)) Global Forecast System 
+    ([GFS](https://www.emc.ncep.noaa.gov/emc/pages/numerical_forecast_systems/gfs.php)) 
+    - is used to link variations in observed XCH4 to potential upwind sources. 
+
+    The sources of variation in observed XCH4 include 1) dispersed area emissions, 
+    2) discrete point sources, 3) inflow across the domain boundary, and 
+    4) the background concentration. Discrete point source emissions are 
+    determined individually using a divergence integral approach described by 
+    [Chulakadaba et al. (2023)](https://egusphere.copernicus.org/preprints/2023/egusphere-2023-822/) 
+    and prescribed to the inverse model. XCH4 inflow across the domain boundary 
+    and dispersed area emissions are then estimated simultaneously using an 
+    inverse model with an enforced non-negative solution. Total emissions are 
+    the sum of dispersed area and point source emissions.
+    
+    This set of initial observations made by MethaneSAT are consistent with 
+    independent empirical data where available from other sources.
+    Contact the data provider for more information about the project at this 
+    link: [https://www.methanesat.org/contact/](https://www.methanesat.org/contact/).
+  |||,
+  keywords: [
+    'climate',
+    'emissions',
+    'ghg',
+    'methane',
+    'edf',
+    'methanesat',
+  ],
+  providers: [
+    ee.producer_provider('Environmental Defense Fund - MethaneSAT', 'https://methanesat.org'),
+    // This is always the last entry.
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2006-01-24T00:00:00Z', '2025-01-17T00:00:00Z'),
+  summaries: {
+    'eo:bands': [
+      {
+        name: 'flux',
+        description: 'Methane emissions traceable to a 1km^2 area.',
+        gsd: 1000,
+        'gee:units': units.kg_per_hour_per_square_km,
+      }
+    ],
+    flux: {minimum: 0, maximum: 28.3, 'gee:estimated_range': true},
+    'gee:visualizations': [
+      {
+        display_name: 'Methane area sources flux in kg/h/km^2',
+        lookat: { lon: -102.5, lat: 31.85, zoom: 8 },
+        image_visualization: {
+          band_vis: {
+            min: [0],
+            max: [18],
+            bands: ['flux'],
+            palette: ['navy', 'magenta', 'orange', 'yellow'],
+          }
+        },
+      },
+    ],
+    'gee:schema': [
+      {
+        name: 'collection_id',
```

Comment:
do you mind sorting these alphabetically?

Reply:
Done